lib/active_merchant/billing/gateways/jackson_river.rb

```diff
@@ -74,6 +74,7 @@ def add_payment(post, payment)
       def add_metadata(post, options = {})
         post[:ms] = options[:market_source] if options[:market_source]
         post[:guid] = options[:guid] if options[:guid]
+        post[:canvasser_name] = truncate(options[:canvasser_name], 100) if options[:canvasser_name]
```

Comment:
Curious as to where this `truncate` method is coming from?

Reply:
@pnicholls https://github.com/waysact/active_merchant/blob/bf89a6ff287838f8476b79432019055022fd4949/lib/active_merchant/billing/gateway.rb#L294-L297